Offseason2022/src/main/java/frc/robot/commands/ClimberSetGatehook.java

```diff
@@ -4,22 +4,28 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
 
 /**
  *
  */
 public class ClimberSetGatehook extends CommandBase
 {
-  public ClimberSetGatehook( )
+  private boolean m_climberGateHookClosed;
+
+  public ClimberSetGatehook(boolean climberSetGateHook)

```

Comment:
An improvement for readability in variable naming here. When ClimberSetGateHook() is being called, the caller is requesting that the gate hook be put in one of two states--open or closed. Naming the boolean parameter something like "gateHookClosed" and passing in true or false would be easier for the reader to understand. And change the m_ variable to match it.